build/push-okd4.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+validateOC() {
+  OC_USER_NAME=$(oc whoami 2>/dev/null)
```

Comment:
Strange problem here for me:
```
$ oc whoami 
kube:admin
```
but 
```
docker login -u kube:admin
```
fails and
```
docker login -u kubeadmin
```
works

Reply:
@rigazilla will update with solution provided by Pavel